packages/components/src/RecurringSelect/components/DayOfMonthSelect/DayOfMonthSelect.css

```diff
@@ -0,0 +1,84 @@
+/* 
+  faking a border with a background on the parent element, gaps & padding
+  to prevent double borders on overlapping boxes
+*/
+.container {
+  display: grid;
+  width: 100%;
+  padding: var(--space-smaller);
+  border: var(--border-base) solid var(--color-border);
+  border-radius: var(--radius-large);
+  grid-template-areas:
+    ". . . . . . ."
+    ". . . . . . ."
+    ". . . . . . ."
+    ". . . . . . ."
+    ". . . lastday lastday lastday lastday";
+  row-gap: var(--space-smaller);
+  column-gap: var(--space-smaller);
+}
+
+.checkboxWrapper {
+  position: relative;
+}
+
+.checkboxWrapper:nth-child(n + 1):nth-last-child(-n + 1) {
+  grid-area: lastday;
+}
+
+/* kept present but hidden for accessibility */
+.container input[type="checkbox"] {
```

Comment:
For standards sake, can we follow the more common practice of CSS clip for this? (reference from [WebAIM](https://webaim.org/techniques/css/invisiblecontent/))

### CSS clip

```
{clip: rect(1px, 1px, 1px, 1px);
clip-path: inset(50%);
height: 1px;
width: 1px;
margin: -1px;
overflow: hidden;
padding: 0;
position: absolute;}
```

> This fairly modern technique will hide or clip content that does not fit into a 1-pixel visible area. Like off-screen content, it will be visually hidden but still readable by modern screen readers.